src/util/metric/mod.rs

```diff
@@ -96,6 +97,9 @@ impl NonblockUdpMetricSink {
 
 impl MetricSink for NonblockUdpMetricSink {
     fn emit(&self, metric: &str) -> io::Result<usize> {
-        self.socket.send_to(metric.as_bytes(), &self.sink_addr)
+        let t = SlowTimer::from_millis(500);
+        let r = self.socket.send_to(metric.as_bytes(), &self.sink_addr);
+        slow_log!(t, "send metric too slow, takes {:?}", t.elapsed());
```

Comment:
I don't think it's a good idea to measure time costs here. The send operation is usually fast and executed very frequently.

Maybe a bench of `SlowTimer` is needed.